source/app/blueprints/case/case_routes.py

```
@@ -0,0 +1,456 @@
+
+@socket_io.on('save')
+@ac_socket_requires(CaseAccessLevel.full_access)
+def socket_summary_onsave(data):
+
+    data['last_saved'] = current_user.user
+    emit('save', data, to=data['channel'], skip_sid=request.sid)
+
+
+@socket_io.on('clear_buffer')
+@ac_socket_requires(CaseAccessLevel.full_access)
+def socket_summary_onchange(message):
```

Ruff (0.8.2)

Redefinition of unused `socket_summary_onchange` from line 157

(F811)